lib/user-interface/react/src/components/model-management/create-model/BaseModelConfig.tsx

```diff
@@ -128,6 +128,20 @@ export function BaseModelConfig (props: FormProps<IModelRequest> & BaseModelConf
                         checked={props.item.streaming}
                     />
                 </FormField>
+                <FormField label='Image Input' errorText={props.formErrors?.imageInput}>
+                    <Toggle
+                        onChange={({ detail }) => {
+                            if (detail.checked && props.item.features.filter((feature) => feature.name === 'imageInput').length === 0) {
+                                props.setFields({'features': props.item.features.concat({name: 'imageInput', overview: ''})});
+                            } else if (!detail.checked && props.item.features.filter((feature) => feature.name === 'imageInput').length > 0) {
```

Comment:
Same as above, just `!== undefined` (or we could just drop that all together and let JS run wild and free)